.github/workflows/test-B01_SL_load_single_file.yml

```
@@ -24,4 +24,6 @@ jobs:
       - name: first step B01_SL_load_single_file
         run: python src/icesat2_tracks/analysis_db/B01_SL_load_single_file.py 20190502052058_05180312_005_01 SH_testSLsinglefile2 True
       - name: second step make_spectra
-        run: python src/icesat2_tracks/analysis_db/B02_make_spectra_gFT.py SH_20190502_05180312 SH_testSLsinglefile2 True
\ No newline at end of file
+        run: python src/icesat2_tracks/analysis_db/B02_make_spectra_gFT.py SH_20190502_05180312 SH_testSLsinglefile2 True
+      - name: third step make_spectra
+        run: python src/icesat2_tracks/analysis_db/B03_plot_spectra_ov.py SH_20190502_05180312 SH_testSLsinglefile2 True
```

```suggestion
        run: python src/icesat2_tracks/analysis_db/B03_plot_spectra_ov.py SH_20190502_05180312 SH_testSLsinglefile2 True
   
```

Maybe formatters/pre-commit can be configured to add a new line at the end of files?

---

let me add the formatter in another PR

---

This might be a local setting, though.